qt_gui_app/CMakeLists.txt

```diff
@@ -4,7 +4,8 @@ project(qt_gui_app)
 find_package(ament_cmake REQUIRED)
 find_package(ament_cmake_python REQUIRED)
 
-install(PROGRAMS scripts/qt_gui_app DESTINATION bin)
+install(PROGRAMS scripts/qt_gui_app
+  DESTINATION lib/${PROJECT_NAME})
```

Comment:
This is unrelated to "Adding required libraries".

Reply:
I'll make a separate PR